notebooks/Imputation_best_practices/Imputation_best_practices.ipynb

```diff
@@ -0,0 +1,4557 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    from itertools import chain,cycle
</pre><p>Could you please move all imports to the top of the notebook in the first cell?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/data-science-workflows/pull/113/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>

Reply:
<p>In my opinion it is better here, because it will give a context of why the import is being used and for what purpose. Any harm if I keep it here ?</p>